configs/components/cpp-pcp-client.rb

```diff
@@ -11,7 +11,12 @@
     pkg.environment "PATH", "#{settings[:bindir]}:/opt/pl-build-tools/bin:$(PATH)"
   end
 
-  pkg.build_requires "openssl"
+  if !settings[:vendor_openssl]
+    pkg.build_requires 'openssl-devel'
```

Comment:
Since this is primarily meant for allowing PA for FIPS, I will have limit to just RHEL family. It will fail for other OS's. Unless we want it to silently ignore the environment variable.